components/automate-deployment/pkg/converge/converger.go

```diff
@@ -444,12 +447,11 @@ func (t *timeoutable) TimeoutChan() <-chan time.Time {
 	return t.timer.C
 }
 
-func (t *timeoutable) StopTimer(timedout bool) {
+func (t *timeoutable) StopTimer() {
 	if t.timer == nil {
 		return
 	}
-
-	if !t.timer.Stop() && !timedout {
+	if !t.timer.Stop() {
 		<-t.timer.C
 	}
```

Comment:
We could consider setting `t.timer` to `nil` here to avoid possibly trying to Stop it again.